Form/RequestPasswordType.php

```diff
@@ -22,14 +22,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'userName',
                 TextType::class,
                 [
-                    'label' => 'sumocoders.multiuserbundle.form.user',
-                    'required' => true,
+                    'attr' => [
+                        'class' => 'input-lg'
```

Comment:
hoezo? je kan toch {{ form_widget(form.userName, {attr: {class: 'input-lg' }) }} doen? dit heeft toch niks met je form te maken, hoort volgens mij in je template